website/cf-worker/index.js

```diff
@@ -0,0 +1,90 @@
+import {
```

Comment:
Can you move this elsewhere? as this stands this will be included in the Statiq output because it's in the input folder.